bundles/AdminBundle/Controller/Admin/TagsController.php

```diff
@@ -133,7 +133,7 @@ public function treeGetChildrenByIdAction(Request $request)
         if (!empty($request->get('filter'))) {
             $filterIds = [0];
             $filterTagList = new Tag\Listing();
-            $filterTagList->setCondition('`name` LIKE '. $filterTagList->quote('%'. $request->get('filter') .'%'));
+            $filterTagList->setCondition('`name` LIKE '. $filterTagList->quote('%'. strtolower($request->get('filter')) .'%'));
```

Comment:
```suggestion
            $filterTagList->setCondition('`name` LIKE '. $filterTagList->quote('%'. mb_strtolower($request->get('filter')) .'%'));
```

But shouldn't `LIKE` be case-insensitive anyway when Pimcore tables are `utf8mb4_*_ci`?
https://mariadb.com/kb/en/like/
https://dev.mysql.com/doc/refman/5.7/en/case-sensitivity.html

Reply:
@NiklasBr https://github.com/pimcore/pimcore/blob/9a1bde84311eb9aa02fbd00b1d7c13a67ce6e993/bundles/InstallBundle/Resources/install.sql#L494

Reply:
The column `name` is `utf8_bin` because of the uniqueness of a name, but of course for the search it makes sense to be case-insensitive.